### PR TITLE
Fixing miniconda path in .travis.xml and updating helpers 

### PR DESCRIPTION
**Pull request by bsipocz**
Miniconda's default prefix changed recently causing all travis builds failing.

And while I was at it I've updated the helpers to the latest one.

### CODE REVIEW

**Comment by bsipocz** [comment]
@astrofrog - Do you want to use ``ci-helpers`` here, too?

**Comment by astrofrog** [comment]
@bsipocz - yes, that would be great, thanks!

File: .travis.yml [hunk 1]
```diff
@@ -12,22 +12,22 @@ addons:
...
-    # This is just for "egg_info".  All other builds are explicitly given in the matrix
+    - 3.5
+
 env:
     global:
         # The following versions are the 'default' for tests, unless
         # overidden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
-        - NUMPY_VERSION=1.9
+        - NUMPY_VERSION=1.10
```
Comment (astrofrog) [inline .travis.yml]: @bsipocz - can you change this to ``stable``?

Comment (bsipocz) [inline .travis.yml]: sure.

**Comment by bsipocz** [comment]
Some of the tests fail with the OrderedDict issue, and many fail with ``pytest-mpl``. I guess we should fix mpl version to 1.4.3 to get them pass?

**Comment by astrofrog** [comment]
@bsipocz - yes, that might make sense for now. I can look at the pytest-mpl issues today. Can you update the ``OrderedDict`` imports?

**Comment by bsipocz** [comment]
It's already in #184

**Comment by bsipocz** [comment]
Do you want appveyor, too?

**Comment by astrofrog** [comment]
@bsipocz - I'm not sure if the image testing will work on Windows, so maybe we can do that in another PR once one of us has time

**Comment by bsipocz** [comment]
Ohh, sure, I've already forgot about the image testing issues. That case it should be definitely a new PR.

**Comment by bsipocz** [comment]
@astrofrog - now I've rebased this on #184, so it should pass and be ready for review.

**Comment by astrofrog** [comment]
@bsipocz - I think the remaining issue is because you need to change the language to c and set ``PYTHON_VERSION`` manually (like we do in the core package). Otherwise it tries to get a built-in 3.5, which doesn't seem to work.

**Comment by bsipocz** [comment]
@astrofrog - Could you please restart that build? It didn't fail before, and the last commit didn't touch any 3.5 related stuff so I think it's very probably just a random fail.

File: wcsaxes/frame.py [hunk 2]
```diff
@@ -2,9 +2,17 @@
...
 import abc
 
+# This is for python 2.6
+try:
+    from collections import OrderedDict
+except ImportError:  # pragma no cover
+    try:
+        from ordereddict import OrderedDict
+    except ImportError:
+        OrderedDict = dict
```
Comment (astrofrog) [inline wcsaxes/frame.py]: You can actually just remove Python 2.6 support and include only the import from ``collections``

Comment (bsipocz) [inline wcsaxes/frame.py]: OK, sounds good.

**Comment by astrofrog** [comment]
Thank you very much @bsipocz! :)

**Pull request merged**